aten/src/ATen/native/native_functions.yaml

```diff
@@ -9312,6 +9308,7 @@
   dispatch:
     CPU: max_pool2d_with_indices_out_cpu
     CUDA: max_pool2d_with_indices_out_cuda
+    QuantizedCPU: max_pool2d_with_indices_out_quantized_cpu
```

Comment:
can we add a test for `max_pool2d_with_indices` in test_quantized_ops.py? existing test is here: https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_op.py#L1222